Process.c

```diff
@@ -1026,8 +1026,7 @@ const ProcessClass Process_class = {
       .delete = Process_delete,
       .compare = Process_compare
    },
-   .writeField = Process_writeField,
-   .getCommandStr = Process_getCommandStr,
+   .writeField = Process_writeField
```

Comment:
```suggestion
   .writeField = Process_writeField,
```

Why not keep this comma?